test/e2e/clusterctl_upgrade_test.go

```diff
@@ -314,3 +314,51 @@ var _ = Describe("When testing clusterctl upgrades using ClusterClass (v1.6=>cur
 		}
 	})
 })
+
+var _ = Describe("When testing clusterctl upgrades (v1.7=>current)", func() {
+	// Get v1.7 latest stable release
+	version := "1.7"
+	stableRelease, err := GetStableReleaseOfMinor(ctx, version)
+	Expect(err).ToNot(HaveOccurred(), "Failed to get stable version for minor release : %s", version)
+	ClusterctlUpgradeSpec(ctx, func() ClusterctlUpgradeSpecInput {
+		return ClusterctlUpgradeSpecInput{
+			E2EConfig:                 e2eConfig,
+			ClusterctlConfigPath:      clusterctlConfigPath,
+			BootstrapClusterProxy:     bootstrapClusterProxy,
+			ArtifactFolder:            artifactFolder,
+			SkipCleanup:               skipCleanup,
+			InfrastructureProvider:    ptr.To("docker"),
+			InitWithBinary:            fmt.Sprintf(clusterctlDownloadURL, stableRelease),
+			InitWithProvidersContract: "v1beta1",
+			//  Note: Both InitWithKubernetesVersion and WorkloadKubernetesVersion should be the highest mgmt cluster version supported by the source Cluster API version.
+			InitWithKubernetesVersion: "v1.29.2",
+			WorkloadKubernetesVersion: "v1.29.2",
+			MgmtFlavor:                "topology",
+			WorkloadFlavor:            "",
+		}
+	})
+})
+
+var _ = Describe("When testing clusterctl upgrades using ClusterClass (v1.7=>current) [ClusterClass]", func() {
+	// Get v1.6 latest stable release
+	version := "1.7"
```

Comment:
Is this one supposed to be `"1.6"`?

Reply:
It should be `"1.7"`, I just made a mistake while writing the comment!